ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -151,6 +158,24 @@ chain:
       oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
       echo "Cluster became available, creating kubeconfig"
       bin/hypershift create kubeconfig --namespace=clusters --name=${CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+      
+      # In case of BYO CNI, the HC will never be ready before CNI components are up and running
+      if [[ $HYPERSHIFT_NETWORK_TYPE == "Other" || $HYPERSHIFT_NETWORK_TYPE == "Calico" ]]; then
```

Comment:
is it same for calico cni?

Reply:
Yes. Calico has to be deployed manually as well.

Reply:
/lgtm